tests/expose_test.go

```diff
@@ -428,11 +424,10 @@ var _ = Describe("[rfe_id:253][crit:medium][vendor:cnv-qe@redhat.com][level:comp
 				virtctl := tests.NewRepeatableVirtctlCommand(getVirtctlExposeCommand(vmiExposeArgs, ipFamily)...)
 				Expect(virtctl()).ToNot(HaveOccurred())
 
-				By("Getting back the cluster IP given for the service")
-				svc, err := virtClient.CoreV1().Services(udpVM.Namespace).Get(serviceName, k8smetav1.GetOptions{})
-				Expect(err).ToNot(HaveOccurred())
-				serviceIP := svc.Spec.ClusterIP
-				nodePort := svc.Spec.Ports[0].NodePort
+				serviceIP, err := getServiceIP(virtClient, udpVM.GetNamespace(), serviceName)
+				Expect(err).ToNot(HaveOccurred(), "should have been able to retrieve the service IP")
+
+				nodePort, err := getServiceNodePort(virtClient, udpVM.GetNamespace(), serviceName)
 				Expect(nodePort).To(BeNumerically(">", 0))
```

Comment:
Expect(err).ToNot(HaveOccurred(), "should have been able to retrieve the node port")